InvenTree/common/models.py

```diff
@@ -1664,6 +1664,10 @@ class Meta:
         help_text=_('Unit price at specified quantity'),
     )
 
+    updated = models.DateTimeField(
```

Comment:
@nwns I think I understand the issues: There should be the option for empty entries so that already existing entries are still valid. So something like the following should be added (please Errol the migrations after that):

```
blank=true,
null=true,
```

Reply:
I don't think this should be nullable.  There is always a last update.  Also, the migrations put the current time in if the field was NULL.